src/libman/Contracts/ConsoleLogger.cs

```diff
@@ -35,6 +35,29 @@ public string GetUserInput(string fieldName)
             }
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="fieldName"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        public string GetUserInputWithDefault(string fieldName, string defaultValue)
+        {
+            lock (_syncObject)
+            {
+                string message = $"{fieldName} [{defaultValue}]: <enter>";
```

Comment:
><enter> [](start = 65, length = 7)

remove

Reply:
:thumbsup: 

---
In reply to: [195176822](https://github.com/aspnet/LibraryManager/pull/158#discussion_r195176822) [](ancestors = 195176822)